docs/dev-guide/integration/events/index.rst

```diff
@@ -9,15 +9,46 @@ status. This is enabled by setting ``event_notifications_enabled`` under
 ``event_notification_url`` may be set if the notification AMQP server is not
 running locally on port 5672.
 
+.. warning::
+   It is highly recommended to use a version of ``python-kombu`` that includes
```

Comment:
I'll suggest a rewrite to shorten this and make it more declarative.

```
It is highly recommended to use ``python-kombu-3.0.24-6.pulp`` or newer which includes a fix for a connection leak.
```

I don't think linking to the issue is that important. Interested users can always look at the changelog for python-kombu to have a trail leading them to the code.